<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Системная информация</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="2">
             <widget class="QLabel" name="userName">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLabel" name="_fastSwitch">
              <property name="text">
               <string>Быстрое переключение задач:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="pcName">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLabel" name="mouseInfo">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="_sysDir">
              <property name="text">
               <string>Путь к системному каталогу: </string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="winDir">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="sysParams">
              <property name="text">
               <string>Cистемные параметры</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="_pcName">
              <property name="text">
               <string>Имя компьютера: </string>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QLabel" name="fastSwitch">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="winVersion">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="10" column="2">
             <widget class="QLabel" name="soundSentry">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="tempDir">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="sysDir">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="_winDir">
              <property name="text">
               <string>Путь к каталогу Windows: </string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="_winVersion">
              <property name="text">
               <string>Версия Windows:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="_tempPath">
              <property name="text">
               <string>Путь к каталогу временных файлов: </string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="_mouseInfo">
              <property name="text">
               <string>Настройки мыши:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="_userName">
              <property name="text">
               <string>Имя пользователя: </string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLabel" name="_soundSentry">
              <property name="text">
               <string>При изменении системных настроек</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Системные метрики</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="sysMetrics">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Настройки шрифтов и мыши</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="11" column="0" colspan="2">
           <widget class="QCheckBox" name="mouseKeysOn">
            <property name="text">
             <string>Включить управление с помощью клавиатуры</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Наклон символов:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="setDefault">
            <property name="text">
             <string>По умолчанию</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="_fontHeight">
            <property name="text">
             <string>Высота шрифта:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="fontEscapement"/>
          </item>
          <item row="8" column="0" colspan="3">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QLabel" name="_maxSpeed">
            <property name="text">
             <string>Максимальная скорость курсора</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="fontHeight"/>
          </item>
          <item row="16" column="2">
           <widget class="QSpinBox" name="maxSpeed"/>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Управление мышью с помощью клавиатуры</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="fontWidth"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Толщина линий начертания шрифта:</string>
            </property>
           </widget>
          </item>
          <item row="17" column="2">
           <widget class="QSpinBox" name="mouseCtrlSpeed"/>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="confirmation">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QCheckBox" name="strikeOut">
              <property name="text">
               <string>Перечеркнутое</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="underLine">
              <property name="text">
               <string>Подчеркнутое</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="italic">
              <property name="text">
               <string>Курсив</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="fontOrientation"/>
          </item>
          <item row="14" column="0" colspan="2">
           <widget class="QCheckBox" name="mouseHotkeySound">
            <property name="text">
             <string>Подавать звуковой сигнал при изменении режима</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0" colspan="2">
           <widget class="QCheckBox" name="mouseConfirmHotkey">
            <property name="text">
             <string>Выводить диалоговое окно для подтверждения</string>
            </property>
           </widget>
          </item>
          <item row="18" column="0">
           <widget class="QPushButton" name="mouseSetDefault">
            <property name="text">
             <string>По умолчанию</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="fontWeight"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Настройки шрифта</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="_fontWidth">
            <property name="text">
             <string>Ширина шрифта:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Угол нанесения шрифта:</string>
            </property>
           </widget>
          </item>
          <item row="15" column="0" colspan="2">
           <widget class="QCheckBox" name="mouseModifiers">
            <property name="text">
             <string>Ctrl - ускорение, Shift - замедление движения</string>
            </property>
           </widget>
          </item>
          <item row="17" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Ускорение</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="isMouseKeysAvaliable">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0" colspan="2">
           <widget class="QCheckBox" name="mouseHotkeyActive">
            <property name="text">
             <string>Разрешить переключение режима MouseKeys по Alt+ Shift+Num Lock.</string>
            </property>
           </widget>
          </item>
          <item row="18" column="1">
           <widget class="QPushButton" name="mouseConfirmation">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
          <item row="19" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Системные цвета</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="2">
           <widget class="QLabel" name="inactivecaptions">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>COLOR_INACTIVECAPTION</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="brighlight">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="scrollbar">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="confirmBrighlight">
            <property name="text">
             <string>Выбрать цвет...</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QPushButton" name="confirmScrollbar">
            <property name="text">
             <string>Выбрать цвет...</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>COLOR_SCROLLBAR</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>COLOR_BRIGHLIGHT</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="confirmInactivecaptions">
            <property name="text">
             <string>Выбрать цвет...</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="2">
           <widget class="QPushButton" name="setDefaultColors">
            <property name="text">
             <string>По умолчанию</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Страница</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout">
          <item row="5" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Текущее смещение (относительно UTC):</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="currentBias">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Величина стандартного смещения:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="standartBias">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Величина смещения после перевода часов:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="daylightBias">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Системное время</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateTimeEdit" name="systemTime"/>
          </item>
          <item row="2" column="1">
           <widget class="QDateTimeEdit" name="localTime"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Локальное время</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Страница</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Другие метрики</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Код последней ошибки:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lastError">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Очередь сообщений вызывающего потока</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="inpState">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QTableWidget" name="codePages"/>
          </item>
          <item row="0" column="1">
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
